app/actions/CompanyInterestActions.ts

```diff
@@ -1,9 +1,10 @@
-import { CompanyInterestForm } from './ActionTypes';
+import type { Thunk } from 'app/types';
+import type { CompanyInterestEntity } from 'app/reducers/companyInterest';
+import { companyInterestSchema } from 'app/reducers';
 import callAPI from 'app/actions/callAPI';
 import { addToast } from 'app/actions/ToastActions';
-import { companyInterestSchema } from 'app/reducers';
-import type { CompanyInterestEntity } from 'app/reducers/companyInterest';
-import type { Thunk } from 'app/types';
+import { CompanyInterestForm } from './ActionTypes';
```

Comment:
Do we not want type imports to come last?

Reply:
So, right now, type imports are between external (npm) modules and internal imports (app/...).
I thought that would be nice since then the type imports are sort of inbetween internal and external things, considering we import types from both groups.

Reply:
hmm, okay, I see your point. I'm just used to putting them at the bottom. I'll let you decide then 👍🏼 Perhaps somebody else got an input on this?

Reply:
People at the office agreed with types at the bottom, so I'll do that :+1: